static/app/views/performance/traces/content.tsx

```diff
@@ -399,6 +402,10 @@ function useTraces<F extends string>({
       sort,
       per_page: limit,
       maxSpansPerTrace: 10,
+      // breakdownCategory: ['db'],
+      breakdownCategory: [],
+      minBreakdownDuration: 2,
+      minBreakdownPercentage: 1 / 40,
```

Comment:
Did we want to keep the min breakdown duration and percentage now that we're not using the categories?

Reply:
Nah I'll remove em